spec/jobs/state_file/send_reject_resolution_reminder_notification_job_spec.rb

```diff
@@ -24,35 +25,72 @@
       allow(StateFile::MessagingService).to receive(:new).with(intake: intake, message: message, body_args: body_args).and_return(sf_messaging_service)
     end
 
-    context "with an intake that has been sent a notified-of-rejection message, does not have an accepted return and is not currently in an in-progress state" do
-      it "sends the message" do
-        expect {
-          described_class.perform_now(intake)
-        }.to change(StateFileNotificationEmail, :count).by(1)
+    context "with an intake that has been sent a notified-of-rejection message, does not have an accepted return" do
+      context "is currently in waiting state" do
+        let(:current_state) { :waiting }
+        before do
+          efile_submissions.first.efile_submission_transitions.first.update(sort_key: 1)
+          create(:efile_submission_transition, :notified_of_rejection, efile_submission: efile_submissions.first, most_recent: false, sort_key: 0)
+        end
+
+        it "sends the message" do
+          expect {
+            described_class.perform_now(intake)
+          }.to change(StateFileNotificationEmail, :count).by(1)
+
+          expect(intake.reload.message_tracker).to include("messages.state_file.reject_resolution_reminder")
+
+          expect(StateFile::MessagingService).to have_received(:new).with(
+            intake: intake,
+            message: message,
+            body_args: body_args)
+
+          # can re-send if message was sent before (send 13th & 23rd in 2025)
+          expect {
+            described_class.perform_now(intake)
+          }.to change(StateFileNotificationEmail, :count).by(1)
 
-        expect(intake.reload.message_tracker).to include("messages.state_file.reject_resolution_reminder")
+          expect(intake.reload.message_tracker).to include("messages.state_file.reject_resolution_reminder")
```

Comment:
should this be expect it to be included twice?

Reply:
it seems to just update the timestamp to the second time it was sent :(